src/routes/+page.svelte

```diff
@@ -4,6 +4,7 @@
 <h2>🦅 American</h2>
 <ul>
   <li><a href="/american/burger">Greasy Hamburger</a></li>
+  <li><a href="/american/pizza">Pizza</a></li>
```

Comment:
Looks good! Let's make the list item the same name as what's on the page itself

```suggestion
  <li><a href="/american/pizza">Fly Pizza</a></li>
```